### PR TITLE
update version

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

File: tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## Version 0.1.0 (06-26-2023)
```
Comment (kristapratico) [inline tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md:3]: Should this be in YYYY-MM-DD format?

File: tools/pylint-extensions/azure-pylint-guidelines-checker/setup.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="azure-pylint-guidelines-checker",
-    version="0.0.8",
+    version="0.1.0",
```
Comment (kristapratico) [inline tools/pylint-extensions/azure-pylint-guidelines-checker/setup.py:9]: Is this the version we want for a bug fix?

File: tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md (hunk 1 above)
Comment (l0lawrence) [reply tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md:3]: sorry I copied over from another tool changelog, yes will fix to match python :)

File: tools/pylint-extensions/azure-pylint-guidelines-checker/setup.py (hunk 2 above)
Comment (l0lawrence) [reply tools/pylint-extensions/azure-pylint-guidelines-checker/setup.py:9]: I was going to start aligning with how python version bumps work with Breaking.Major.Minor, I thought a bug fix could be major, but can make it minor? So I can make it .0.0.9 then

Comment (kristapratico) [reply tools/pylint-extensions/azure-pylint-guidelines-checker/setup.py:9]: semvar uses MAJOR.MINOR.PATCH. This feels like a patch to me.

File: tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## Version 0.0.9 (2023-06-26)
```
Comment (kristapratico) [inline tools/pylint-extensions/azure-pylint-guidelines-checker/CHANGELOG.md:3]: I don't think we normally put "Version" in the header, so as long as no tool is parsing this it's probably okay

**Review by kristapratico — APPROVED** [review]

**Pull request merged**